src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -372,59 +362,59 @@
 
         "deletedLength": 
         {
-          "description": "An optional integer specifying the number of bytes to delete, start at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "Specifies the number of bytes to delete, starting at the byte offset specified by offset, measured from the beginning of the file.",
           "type": "integer",
           "minimum": 1
         },
 
         "insertedBytes": 
         {
-          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the 'offset' property, measured from the beginning of the file.",
           "type": "string"
         }
       },
-      "required": ["offset", "deletedLength", "insertedBytes"]
+      "required": ["offset"]
     },
     "result": 
     {
-      "description": "Represents one or more observations about an analysis target produced by a static analysis tool.",
+      "description": "A single result produced by a static analysis tool.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
         "ruleId": 
         {
-          "description": "An opaque, stable identifier that uniquely identifies the specific rule associated with the result (e.g., CA2001).",
+          "description": "A stable, opaque identifier for the rule that was evaluated to produce the result.",
```

Comment:
can we just say 'the rule that produced the result'? note that elsewhere, by the way, you claim that it is the location that produces the result, not the rule. 'the location that produced the result'. so who produced the result? :) i might have said 'the location associated with a result'